### ISSUE TITLE
Location Indicator in Textarea

### ISSUE THREAD

**Issue by dsdoermann** [robindijkhof/overleaf-textarea#80]
At one time, I believe I saw indications of colored text were changes would be applied when using the chrome plugin and overleaf/Grammarly.  But now I see a faint cursor.  Is there some configuration I need to do to see for example, a highlight or underline to know where the edit will be applied?
![Capture](https://user-images.githubusercontent.com/43833343/176255044-42e7ee37-a8aa-48d8-98ee-c430b8950655.PNG)
?

**Comment by PCLVO** [comment]
Downgrade to previous Grammarly Extension version 14.1056.0 on Chrome, 8.899.0 for Firefox (still the latest version). That works.

Link to Chrome extension: [https://www.crx4chrome.com/go.php?p=278461&s=5H5D9NWnR2xgI&l=https%3A%2F%2Ff6.crx4chrome.com%2Fcrx.php%3Fi%3Dkbfnbcaeplbcioakkpcpgfkobkghlhen%26v%3D14.1056.0](https://github.com/robindijkhof/overleaf-textarea/issues/Here)

Btw, the app will be automatically updated. You will have to block the update, check this:
[https://stackoverflow.com/questions/27657617/how-to-disable-google-chrome-extension-autoupdate](https://github.com/robindijkhof/overleaf-textarea/issues/url)

You have to delete the Grammarly app first.

Cheers

**Comment by robindijkhof** [comment]
I don't really get it. Could you elaborate?

**Comment by dsdoermann** [comment]
Sorry that it was not clear.  When I'm using Grammarly with overleaf and the textarea on, I get a popup window showing the proposed change, but in the text area, there is no indication of where the change would be applied (see snippet 1 below, no underline or anytihyng on the text window.  I thought that previously there was.  Was I mistaken?

In snippet 2, there is an underline takes from the MSWord version of Grammarly, showing where the change would be applied.

It is just a little hard to get context without any indication of which text would change.

***@***.***


***@***.***

-----------------------------------
David Doermann
Professor, SUNY Empire Innovation
Professor, Department of Computer Science and Engineering
University at Buffalo

email: ***@***.******@***.***>
website: cse.buffalo.edu/~doermann<http://cse.buffalo.edu/~doermann>
phone: 716-645-1557 (office), 410-493-9043 (mobile)

***@***.***

**Comment by robindijkhof** [comment]
I don't have that problem. Which editor are you using? I have to use the legacy editor with this plugin. although, Grammarly seems to work with the new editor out of the box. That might be the end of this plugin.

**Comment by dsdoermann** [comment]
Im using overleaf with chrome.  Is that what you are asking?

Dave

-----------------------------------
David Doermann
Professor, SUNY Empire Innovation
Professor, Department of Computer Science and Engineering
University at Buffalo

email: ***@***.******@***.***>
website: cse.buffalo.edu/~doermann<http://cse.buffalo.edu/~doermann>
phone: 716-645-1557 (office), 410-493-9043 (mobile)

***@***.***

**Comment by robindijkhof** [comment]
No, that's not it. I got 3 editors. Just 'editor', legacy editor and rich text editor. You do not have these options?
